### PR TITLE
Fixed Short Description and Value for stgHCAfflMapAutoEnrollStatusNotFound

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by MatthewBlanski — APPROVED** [review]
This is a simple change. Approved.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -1284,8 +1284,8 @@
         <categories>Setup</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Recommended fix when Health Check can&apos;t find a mapping&apos;s Auto-Enrollment Role</shortDescription>
```
Comment (xinnanlisf) [inline src/labels/CustomLabels.labels:1287]: should we keep the &apos;? I dont quite remember which one we have been using

Comment (MatthewBlanski) [reply src/labels/CustomLabels.labels:1287]: Metadata api pulls down the &apos;, so I put my faith in that. It has been translated well in the past and is a very specific character.

**Comment by spinkelman** [comment]
Passes QA - Merging - Closing WI

**Pull request merged**